### ISSUE TITLE
open tabs stop being clickable after a while

### ISSUE THREAD

**Issue by roslyph** [minbrowser/min#2465]
- **Min Version:** 1.32.1
- **Operating system:** macOS Sonoma 14.5 (23F79)

## Expected Behavior
Click on tab is performed across the entire toolbar area without any problems and makes active this tab.
## Actual Behavior
Clicking on 3/4 tab area doesn't make active this tab. And some click in one place provokes click in another page area.
![bug with click](https://github.com/user-attachments/assets/03393f21-8cec-44a4-92d9-154e703fb47e)

## To Reproduce
Start browser, open more then 10 tabs, switching between active tabs around yours, and wait it a few minutes

**Comment by PalmerAL** [comment]
Sorry, I have the same version of macOS, and I can't reproduce this. Is there anything different/unusual about your mouse settings? Do you have any third-party window management software installed that could affect this?

I can't tell from your recording - are you in full screen? Does it still happen if you aren't?

**Comment by roslyph** [comment]
No, I haven't any window management software never. This screencast was recorded in fullscreen mode. Situation doesn't change when I exit fullscreen mode and continue to interact with browser on window mode. And all of the time I'm use only touchpad.

I remembered. This problem with window interactivity started on version 1.31.0 and has not been fixed after reinstalling the application. Maybe deleting all the log files and dependencies can fix this problem
